src/styles/utils/variables.scss

```
@@ -0,0 +1,10 @@
+$blockSize: 100px;
+$calendarWidth: 100px * 7 + 10px;
+$startDay: 
+  "mon" 0,
+  "tue" #{$blockSize + 1px},
+  "wed" #{$blockSize * 2 + 2px},
+  "thu" #{$blockSize * 3 + 3px},
+  "fri" #{$blockSize * 4 + 4px},
+  "sat" #{$blockSize * 5 + 5px},
+  "sun" #{$blockSize * 6 + 6px};
```

let's add just numbers here, you can calculate the exact width inside @each rule

```suggestion
$startDay: 
  "mon" 0,
  "tue" 1,
  "wed" 2,
  "thu" 3,
  "fri" 4,
  "sat" 5,
  "sun" 6;
```